client/src/js/services/UserService.js

```diff
@@ -79,4 +80,8 @@ function UserService($http, util) {
     return $http.put('/users/' + id + '/password', data)
     .then(util.unwrapHttpResponse);
   }
+
+  function validatePassword(passwordA, passwordB) {
+    return passwordA && passwordA && passwordA === passwordB;
```

Comment:
Is it supposed to include a length check? It seems like that code has been moved but I can't verify the functionality is equivalent.